File: official/docs/node/v6/shipping-refund/refund.js
const EasyPostClient = require('@easypost/api');

const client = new EasyPostClient(process.env.EASYPOST_API_KEY);

(async () => {
  const shipment = await client.Shipment.retrieve('shp_...');

  const refundedShipment = await client.Shipment.refund(shipment.id);

  console.log(refundedShipment);
})();

File: official/docs/golang/current/shipping-refund/refund.go
package shipping_refund

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v5"
)

func refund() {
	client := easypost.New("EASYPOST_API_KEY")

	shipment, _ := client.RefundShipment("shp_...")

	fmt.Println(shipment)
}
